lib/new_super_codebreaker_2021/game.rb

```diff
@@ -0,0 +1,105 @@
+module NewSuperCodebreaker2021
+  class Game
+    include DBMethods
+    include ShowContent
+    include Validate
+
+    attr_accessor :code
```

Comment:
Use `attr_reader` please

Reply:
U can use stubs for tests

Reply:
Or u can move `generate_code` to separated class and mock it

Reply:
fixed